js/sign/tests/integrity-block-signer_test.js

```diff
@@ -43,16 +68,27 @@ describe('Integrity Block Signer', () => {
     return signer;
   }
 
-  it('accepts only ed25519 type of key.', () => {
-    const keypair = crypto.generateKeyPairSync('ed25519');
-    expect(() =>
-      initSignerWithTestWebBundleAndKeys(keypair.privateKey)
-    ).not.toThrowError();
+  function createTestSuffix(publicKey) {
+    return utils.SignatureType[utils.getSignatureType(publicKey)];
+  }
+
+  it('accepts only selected key types.', () => {
+    for (const validKey of [
+      { keyType: 'ed25519' },
+      { keyType: 'ec', options: { namedCurve: 'secp256k1' } },
+    ]) {
+      const keypairValid = crypto.generateKeyPairSync(
+        validKey.keyType,
+        validKey.options
+      );
+      expect(() =>
+        initSignerWithTestWebBundleAndKeys(keypairValid.privateKey)
+      ).not.toThrowError();
+    }
 
     for (const invalidKey of [
       { keyType: 'rsa', options: { modulusLength: 2048 } },
       { keyType: 'dsa', options: { modulusLength: 1024, divisorLength: 224 } },
-      { keyType: 'ec', options: { namedCurve: 'secp256k1' } },
```

Comment:
nit: Maybe keep this but change the `namedCurve` to `sect239k1`?

Reply:
Done!